script/Ciel/Day 12/12_01_CIEL12_2B.txt

```diff
@@ -31,7 +31,7 @@ Not only do I know the truth behind those incidents,
 -- Page 28, Offset 30936.
 -- ……情けない話だけど、どんなに自分を鼓舞したところで先輩がいないという事実が、重く肩にのしかかって――――
 I know it's pathetic, but no matter how much I try to cheer myself up,
- the fact that Senpai is gone still weights heavy on my hea―――
+ the fact that Senpai is gone still weights heavy on my shou―――
```

Comment:
I'd opt to keep this as heart. More appropriate for an emotional toll like this; something like responsibility would be more appropriate for shoulders.

Reply:
Also prefer to keep it as heart. Weights should be amended to Weighs